metricbeat/module/system/process_summary/process_summary.go

```diff
@@ -0,0 +1,97 @@
+// +build darwin freebsd linux windows
+
+package process_summary
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+	"github.com/elastic/beats/metricbeat/module/system/process"
+	"github.com/pkg/errors"
+
+	sigar "github.com/elastic/gosigar"
+)
+
+// init registers the MetricSet with the central registry.
+// The New method will be called after the setup of the module and before starting to fetch data
+func init() {
+	if err := mb.Registry.AddMetricSet("system", "process_summary", New, parse.EmptyHostParser); err != nil {
```

Comment:
What if the metricset name were `process.summary` instead of `process_summary`, would that alleviate some of the concerns about the name differing from the data model?